### PR TITLE
downgrade downloaded objs print

### PR DESCRIPTION
**Pull request by longbowlu**
downloaded obj prints are very noisy

### CODE REVIEW

File: crates/sui-core/src/gateway_state.rs [hunk 1]
```diff
@@ -693,16 +693,25 @@ where
...
+                {
+                    self.store.insert_object_direct(*object_ref, object).await?;
+                }
+                trace!(?object_id, "Downloaded object details: {}", result);
+            }
+            ObjectRead::Deleted(object_ref) => {
+                debug!(?object_ref, "Download object result: object is deleted");
+            }
+            ObjectRead::NotExists(_) => {
+                debug!(?object_id, "Download object result: object does not exist");
```
Comment (lxfind) [inline crates/sui-core/src/gateway_state.rs:712]: This seems to just make it more noisy?

Comment (lxfind) [reply crates/sui-core/src/gateway_state.rs:712]: We already have: https://github.com/MystenLabs/sui/blob/87d5950c06dfc8cd8931ffe89ebcce4cd8c9060d/crates/sui-types/src/object.rs#L620

Comment (longbowlu) [reply crates/sui-core/src/gateway_state.rs:712]: No, it's less. 

We use to `debug!` for either of the three cases. And the thing we dislike the most is when it exists. With this change, we still `debug!` for all cases but when it exists it prints much fewer things

Comment (lxfind) [reply crates/sui-core/src/gateway_state.rs:712]: Maybe I am missing something, but it's supposed to only print the object ref when the object exists (https://github.com/MystenLabs/sui/blob/87d5950c06dfc8cd8931ffe89ebcce4cd8c9060d/crates/sui-types/src/object.rs#L630)

Comment (lxfind) [reply crates/sui-core/src/gateway_state.rs:712]: Note that we used to print with "{:?}" which is why it's displaying the content.
https://github.com/MystenLabs/sui/pull/3358 fixed it and now it's "{}" which will call the Display implementation.

**Review by lxfind — CHANGES_REQUESTED** [review]
We shouldn't need this change

File: crates/sui-core/src/gateway_state.rs (hunk 1 above)
Comment (velvia) [reply crates/sui-core/src/gateway_state.rs:712]: Looking at the code, I believe @lxfind is right.  The `{}` of `ObjectRead` should print out only the ref, and not the contents. Thus for object exists, the change makes no real difference, other than that at `trace` level we can get the details.

**Pull request closed without merging**